packages/plugin-inline/src/Format.ts

```diff
@@ -33,11 +33,9 @@ export class Format extends Modifier {
 
     render(): Node {
         const node = document.createElement(this.htmlTag);
-        const attributes = this.modifiers.find(Attributes);
-        if (attributes) {
-            for (const name of attributes.keys()) {
-                node.setAttribute(name, this.modifiers.find(Attributes).get(name));
-            }
+        const attributes = this.modifiers.get(Attributes);
```

Comment:
The `find` was correct here as well. Basically `get`'s purpose was only for cases where we were doing this kind of stuff:
```typescript
let modifier = node.modifiers.find(...);
if (!modifier) {
    modifier = new Modifier(...);
    node.modifiers.append(modifier);    
}
...
```
Other places should still use `find` to avoid the side-effects of `get`.